woocommerce/woocommerce-gateway-stripe
# Add onboarding wizard UI

# Changes proposed in this Pull Request:

Issue: Link to the GitHub issue this PR addresses (if appropriate).
Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1683

## Description: 
Adds the UPE onboarding wizard to `admin.php?page=wc_stripe-onboarding_wizard`. This PR layout the UI but it doesn't actually store any data to our data store, this will be done in another ticket. Most of the work here are borrowed from https://github.com/Automattic/woocommerce-payments/tree/develop/client/additional-methods-setup. I added components and packages we used in WCPay respectively.

# Testing instructions
1. Make sure you have `_wcstripe_feature_upe_settings` feature flag set to `yes`. Or have [this function](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/9fea8aba02e04b1dc139fa7f46814e73295c3ae0/includes/class-wc-stripe-feature-flags.php#L35) returns true.
2. Go to `wp-admin/admin.php?page=wc_stripe-onboarding_wizard` and you should see the onboarding wizard "Step 1". You should be able to click through to "Step 2", "Step 3":
 
![image](https://user-images.githubusercontent.com/572862/131415591-afdbde89-a492-46ca-ac9b-4dcc111601b3.png)
![image](https://user-images.githubusercontent.com/572862/131415619-c9303b26-5a2c-4633-b2df-45f98a236a05.png)
![image](https://user-images.githubusercontent.com/572862/131415811-d9342cd0-3e6d-4aa8-80f1-39e5289ddae6.png)

4. On "Step 3", clicking "Go to payments settings" will lead you to WC Stripe's settings page `page=wc-settings&tab=checkout&section=stripe`.
